src/procedures/proc_procedures.c

```diff
@@ -0,0 +1,89 @@
+/*
+* Copyright 2018-2020 Redis Labs Ltd. and Contributors
+*
+* This file is available under the Redis Labs Source Available License Agreement
+*/
+
+#include "proc_procedures.h"
+#include "../util/arr.h"
+#include "rax.h"
+#include "procedure.h"
```

Comment:
General formatting comment:
Order the include statement according to line length